src/components/common/FilterGroup/filters/FilterSingleSelection.tsx

```diff
@@ -69,30 +69,12 @@ export default function SingleSelection(props: Props): JSX.Element {
 
   return (
     <div id={`filter-list-${props.field}`} className={classes.box}>
-      <List>
-        {options.map(({ label, value, disabled }) => (
-          <ListItem
-            button
-            key={label}
-            onClick={() => handleChange(value)}
-            selected={props.values.includes(value)}
-            disabled={disabled}
-          >
-            <ListItemText id={value ?? ''} primary={formatLabel(label)} />
-          </ListItem>
-        ))}
-      </List>
+      <Dropdown
+        options={options.map(({ label, value }) => ({ label, value } as DropdownItem))}
+        onChange={(val) => handleChange(val)}
+        selectedValue={props.values[0]}
```

Comment:
one question, should this prop should have one value and not multiple?

Reply:
Should be a single value, yes.

Reply:
Though it's not enforced 🤔 hm, i can take a look at that

Reply:
Alright, i have changed this (and Search) to only take a single value, since they already assume that.